src/notification.cc

```diff
@@ -231,6 +231,27 @@ bool notification::sent_to(const std::string& user) const {
          _notified_contact.end();
 }
 
+/**
+ * @brief insert contacts notified.
+ *
+ * @param contact_notified The names of users notified.
+ */
+
+void notification::insert_contacts(std::set<std::string> contact_notified) {
+  for (auto contact : contact_notified) {
+    _notified_contact.insert(contact);
+  }
```

Comment:
Here, there is a method that does the job for you !
_notified_contact.insert(contact_notified.begin(), contact_notified.end());